dora/tests/src/test/java/alluxio/server/ft/journal/raft/EmbeddedJournalIntegrationTestTransferLeadership.java

```diff
@@ -77,34 +77,6 @@ public void repeatedTransferLeadership() throws Exception {
     mCluster.notifySuccess();
   }
 
-  @Test
-  public void transferWhenAlreadyTransferring() throws Exception {
```

Comment:
In our new implementation what does Ratis do if you issue a transfer when a transfer is happening?

Reply:
Since the `transferLeadership` method is `synchronized` and now processes the transfer synchronously (instead of asynchronously with a `Thread` like before), sequential leadership transfers cannot collide, they will simply be process one after the other.